scripts/generateMaintenanceTestData.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+header="R\"=("
+footer=")=\""
+
+outfile=Plan.json
+echo $header > $outfile
+curl -s [::]:4001/_api/agency/read -d'[["/arango/Plan"]]'|jq .[0].arango.Plan >> $outfile
```

Comment:
use "localhost" instead of [::] so code works with both ipv4 and ipv6.  Might want to add "-l" option so that curl follows redirect.

Reply:
hmm, "localhost:4001" should be default if nothing given on command line.  That way user can slam in an address of an executing integration test and pull that info.

Reply:
localhost added.